Algorithmia/algorithm.py

```diff
@@ -17,6 +21,10 @@ def __init__(self, client, algoRef):
         if m is not None:
             self.client = client
             self.path = m.group(1)
+            self.username = self.path.split("/")[0]
```

Comment:
Is it safe to assume here that the algorithm URI structure will always remain the same? Is there a world where this is brittle(supporting unicode algorithm names/usernames?)

Reply:
I would assume that if the URI structure ever changes, line 19 would also break (which has been part of the client forever now).

Other than that point, I don't think we'll ever change it.